lib/browser/platform/ios.rb

```diff
@@ -3,10 +3,21 @@ module Browser
   class Platform
     class IOS < Base
       MATCHER = /(iPhone|iPad|iPod)/
-      VERSION_MATCHER = /OS ([\d.]+)/
+      VERSION_MATCHER = /OS ((?<major>\d)_(?<minor>\d)_?(?<patch>\d)?)/
 
       def version
-        ua[VERSION_MATCHER, 1] || "0"
+        matches = VERSION_MATCHER.match(ua)
```

Comment:
Yeah, I can change that. 

Skipping MINOR because we want "9" when the actual version is "9.0". The original code was only matching on single number (the regex matched "9" when it was "9_0_1" for example) so I wanted to not break that functionality for backwards compatibility.